README.md

```diff
@@ -45,6 +103,91 @@ This is a simple cookbook to setup a NAT instance, and additionally set source d
     <td>Disable source dest check</td>
     <td><tt>true</tt></td>
   </tr>
+  tr>
+    <td><tt>['aws-vpc-nat-instance']['monitor']['enabled']</tt></td>
+    <td>Boolean</td>
+    <td>Enable monitoring recipe</td>
+    <td><tt>true</tt></td>
+  </tr>
+  tr>
+    <td><tt>['aws-vpc-nat-instance']['monitor']['default_environment_name']</tt></td>
+    <td>String</td>
+    <td>Monitoring recipe relies on special naming convention for NAT instance and route tables names (see the 
+    library source code for details).  The attribute is a mapping between Chef's `_default` environment and AWS
+    environment.</td>
+    <td><tt>prod</tt></td>
+  </tr>
+  tr>
```

Comment:
Fixed in https://github.com/rubikloud/chef-aws-vpc-nat-instance/commit/ac87d07b7714dc3eb397765fa2f39b0e6636ed31

P.S.: browsers are smart enough to filter childish errors like this